examples/cpp/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ function(add_example_with_proving example_target)
 endfunction()
 
 add_example_without_proving(placeholder_verifier_cpp_example SOURCES placeholder_verifier.cpp INPUT placeholder_verifier.inp CURVE_TYPE pallas)
+add_example_without_proving(agg_verifier_cpp_example SOURCES agg_verifier.cpp INPUT agg_verifier_public.inp PRIVATE_INPUT agg_verifier_private.inp CURVE_TYPE pallas)
```

Comment:
we already have `gen_and_check.sh`, `multi_gen_and_check.sh` in CI, why need to add example?